### DISCUSSION TITLE
Issue with lighthouse model connection

### DISCUSSION THREAD

**Discussion by devstevehowe** [nuwave/lighthouse#2581] [General]
I switched from using mysql for some models to mongo, and there should be no issue since it returns an eloquent object the same way, however, for ANY query I am getting "debugMessage": "Found invalid pagination type: paginator", can anyone help figure this out? Running the latest version of lighthouse v6.41.0 and "name": "mongodb/laravel-mongodb", "version": "4.5.0", laravel 11

**Comment by spawnia** [comment]
I am assuming you updated to Lighthouse v6. Use `@paginate(type: PAGINATOR)` as documented, you likely have `type: "paginator"` somewhere.
